### DISCUSSION TITLE
i need this query in lucid. i tried many way but failed

### DISCUSSION THREAD

**Discussion by dmostafiz** [adonisjs/core#3665] [Help]
I need a immediate  help.

SELECT count(*) as count, DATE_FORMAT(created_at,"%Y-%m") as cmonth FROM clicks GROUP BY cmonth

i need this query in lucid ORM. i tried many way but failed. please

**Comment by thetutlage** [comment]
Can you share what you have tried so far?

**Comment by dmostafiz** [comment]
I am just trying to get records from database for certain time like, last 7days, last month, last year etc. 

the table model named Clicks has 2 fields.  *1) user_id *2) created at

its a affiliate system.  if you come to the website using my invitation link then a new click recored will be stored in the clicks table.
 [ database -> clicks -> newRecord( my_id, created_at) ]

![Screenshot 2022-04-07 010526](https://user-images.githubusercontent.com/62952421/162050065-d0010ef4-2680-44ac-8658-adbc6072cda3.png)

![Screenshot 2022-04-07 010419](https://user-images.githubusercontent.com/62952421/162050112-fb91cbbd-3f2f-45a1-a94e-c23c1365fdce.png)
